platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -5,7 +5,6 @@ import { parse as parseSemVer, SemVer, Range } from "semver";
 
 export interface MainManifest {
```

Comment:
One more thing: `semver` is no longer needed, that line and the corresponding dependence can be eliminated (I think it's included at the top level anyway)

Reply:
good catch, thanks!
(maybe here would help an eslint rule for unused code. wdyt ?)

Reply:
good catch, thanks!
(maybe here would help an eslint rule for unused code. wdyt ?)

Reply:
I think that the current linters work at the block or sentence level... I haven't found anything better. If you know of any...

Reply:
I think it's the `'@typescript-eslint/no-unused-vars': ['error', { args: 'none' }],`
there's also the` `'no-unused-expressions': 'error',` `
maybe we can add them on the other PR for eslint stricter config ?

Reply:
I'm afraid it will open a whole can of worms, with lots of other errors. Let's leave that for later maybe, when that one is merged.